### ISSUE TITLE
Exploit in-page nav to make API and CLI commands easier to find

### ISSUE THREAD

**Issue by jessicaschilling** [ipfs/docs#400]
Initial launch of docs beta doesn't include the in-page nav. Let's investigate whether we should bring it back in to make API and CLI commands easier to find until we implement a more robust approach to API/CLI documentation as discussed in #393.

_Note: This issue was generated as a result of Nov 2019 internal user testing. Background can be found in the [summary](https://hackmd.io/gdRqxFZoRy-U-xB1DHitIA?view) (source: #315)._

**Comment by ericronne** [comment]
A sticky in-page nab might not work with such a long list, unfortunately. It would bleed off the bottom. Although it’s no excuse for bad ux, api reference interfaces seem to be almost universally bad. 

If the commands could be chunked into subcategories (a la [Twitter’s api reference](https://developer.twitter.com/en/docs/api-reference-index)), an in-page nav might be helpful.

**Comment by johnnymatthews** [comment]
API docs are different from regular documentation, so it makes sense that we present API stuff differently. [Readme.com](https://docs.readme.com/reference/api-specification) has an exciting way of showing this stuff. It might be that we need to have a completely different template for API docs.

**Comment by ericronne** [comment]
Yes readme's sandbox addition is nice. These references are definitely a sticky design challenge, which few others seem to have handled all that well. In the "All the docsites" hackdoc i've added a separate section to collect samples of good API/CLI reference design.

https://hackmd.io/A0iF9RweSAmZtyanQuKeUw?view#%F0%9F%A4%96-Interesting-API-reference-design. Keep em coming!

**Comment by jessicaschilling** [comment]
Adding screenshot of @ericronne's visual layout of the right-hand nav. 
https://user-images.githubusercontent.com/9259185/68242374-0399b500-ffd6-11e9-8a10-2b5fd21048a3.png
